### PR TITLE
[DOC] Add live trace info

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:
Adds information about the live trace and how it's controlled in the configuration > ingester section. 

Based on https://github.com/grafana/tempo/discussions/2380

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/configuration/_index.md [hunk 1]
```diff
@@ -209,6 +209,11 @@ For more information on configuration options, see [here](https://github.com/gra
 
 The ingester is responsible for batching up traces and pushing them to [TempoDB](#storage).
 
+While a trace is considered live, additional spans can be added.
+A live, or active, trace is a trace that hasn't received a new batch of spans in more than a configured amount of time (default 10 seconds, set by `ingester.trace_idle_period`).
```
Comment (mapno) [inline docs/sources/tempo/configuration/_index.md:213]:
```suggestion
A live, or active, trace is a trace that has received a new batch of spans in more than a configured amount of time (default 10 seconds, set by `ingester.trace_idle_period`).
```

File: docs/sources/tempo/configuration/_index.md [hunk 2]
```diff
@@ -209,6 +209,11 @@ For more information on configuration options, see [here](https://github.com/gra
 
 The ingester is responsible for batching up traces and pushing them to [TempoDB](#storage).
 
+While a trace is considered live, additional spans can be added.
```
Comment (mapno) [inline docs/sources/tempo/configuration/_index.md:212]: It makes it seem as if spans can't be added if the trace isn't live. Wether a trace is live in the ingester is an implementation detail that I'm not sure if it's of a lot of value here.

I think it's better suited when talking about per-tenant limits.

Comment (knylander-grafana) [reply docs/sources/tempo/configuration/_index.md:212]: Okay, what if we just remove that sentence. Would that work? 
```suggestion
```

**Review by zalegrala — APPROVED** [review]

File: docs/sources/tempo/configuration/_index.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/tempo/configuration/_index.md]: Heds confirmed that removing that sentence addressed your concern.

**Pull request merged**